test/common/ws/server_master.js

```diff
@@ -33,7 +33,7 @@ function WSServerMaster() {
 
 	this.headers = WSServerMaster.generatePeerHeaders({
 		ip: this.ip,
-		version: `0.0.${Math.floor(Math.random() * 10) + 1}`,
+		version: '9.8.7',
 	});
```

Comment:
It's better if this is hardcoded and deterministic; it was causing issues and took me ages to find it in the code.